development/ts-migration-dashboard/files-to-convert.json

```diff
@@ -11,8 +11,6 @@
   "app/scripts/controllers/ens/index.test.js",
   "app/scripts/controllers/incoming-transactions.js",
   "app/scripts/controllers/incoming-transactions.test.js",
-  "app/scripts/controllers/metametrics.js",
```

Comment:
We should keep these files here! The name of this file is a bit misleading; it represents files that we determined we ought to convert at the start of the TypeScript migration process. The TypeScript migration dashboard uses this list to know what has been converted. If we take these away, then it can't know that, and the file will no longer show on the dashboard.

Reply:
Oh thanks for that unfortunately we removed all the controllers we worked on recently from this file. I will make an issue or just restore them back in another PR